charts/scalardb/templates/scalardb/deployment.yaml

```diff
@@ -19,6 +19,7 @@ spec:
     metadata:
     {{- if .Values.scalardb.podAnnotations }}
       annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/scalardb/configmap.yaml") . | sha256sum }}
```

Comment:
We can detect the changes of database.properties in the custom values file and reload new database.properties (re-create Pods) automatically using this annotation.

If we change database.properties in the custom values file, the `ConfigMap` is changed and the hash value of `ConfigMap` is changed too.
The Helm detects the changes of hash and re-create Pods automatically.